configure.ac

```diff
@@ -533,6 +533,13 @@ ENABLE_PLUGIN_WITH_TEST(qtglspectrum,
     auto,
     VISUALIZATION)
 
+ENABLE_PLUGIN_WITH_DEP(qthotkey,
```

Comment:
This is wrong because a `QTX11EXTRAS` dependency hasn't been set up in the autoconf script.

Reply:
You're right. Reworked it.